.github/workflows/release.yml

```
@@ -0,0 +1,90 @@
+name: create release
+
+on:
+    push:
+        tags:
+            - 'v*'
```

Any reason not to use the way Bez had it?
```suggestion
            - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```